components/availability/src/Availability.svelte

```diff
@@ -1,13 +1,14 @@
 <svelte:options tag="nylas-availability" />
 
 <script lang="ts">
-  import { ManifestStore } from "../../../commons/src";
+  import { ManifestStore, AvailabilityStore } from "../../../commons/src";
   import { onMount, tick } from "svelte";
   import { get_current_component } from "svelte/internal";
   import { getEventDispatcher } from "@commons/methods/component";
   import type { TimeInterval } from "d3-time";
   import { timeDay, timeHour, timeMinute } from "d3-time";
   import { scaleTime } from "d3-scale";
+  import * as Availability from "@commons/types/Availability";
```

Comment:
Can we import directly instead? Looks like we only need `ClickAction, AvailabilityStatus`. That way we don't need `Availability.AvailabilityStatus.FREE`, etc.

Reply:
We're using 8 of the 9 exports in there though